### PR TITLE
Sign in bugs

### PR DESCRIPTION
**Pull request by Jamil-Bailony**
- [x] fix some issues related to the style in Arabic version (margin between signInWith and hyphen, check box and text)

- [x] extract the validation to a section to use it in sign up too

### CODE REVIEW

File: public/locales/en/translation.json [hunk 1]
```diff
@@ -174,9 +174,22 @@
...
+      "Doctor",
+      "Lawyer",
+      "Psychologist",
+      "Social worker",
+      "All"
+    ],
+    "dropdownTitles": {
+      "location": "Location",
+      "typeOfHelp": "Type of help"
+    }
```
Comment (Jamil-Bailony) [inline public/locales/en/translation.json:192]: from `yarn fix`

**Review by louisrli — APPROVED** [review]

**Review by jennetturkkan — APPROVED** [review]

**Comment by hiba-machfej** [comment]
@Ammar-64

**Comment by Jamil-Bailony** [comment]
@Ammar-64

**Comment by jennetturkkan** [comment]
@Ammar-64 :D

**Comment by Ammar-64** [comment]
There’s a conflicted file here

**Comment by Jamil-Bailony** [comment]
solved @Ammar-64

**Pull request merged**